src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml

```diff
@@ -627,6 +627,10 @@ components:
           type: string
           format: date-time
           x-field-extra-annotation: "@com.fasterxml.jackson.annotation.JsonFormat(pattern=\"yyyy-MM-dd'T'HH:mm:ss'Z'\")"
+        durationHoursTxt:
```

Comment:
why string and not number?

Reply:
This and durationMinutes are being returned as a number now.